src/components/tooltip/tooltip.scss

```diff
@@ -0,0 +1,93 @@
+$z-index-tooltip: 2;
+$z-index-tooltip-closer: 5;
+
+.tooltip {
+  color: #00A9EC;
+  margin-left: 10px;
+  position: relative;
+  display: inline;
+  p {
+    margin: 0;
+  }
+  &-questionmark {
+    width: 16px;
+    height: 16px;
+    cursor: pointer;
+    display: inline;
+    &:focus {
+      outline:0;
```

Comment:
Also hurting accessibility. Better with a custom outline if the original is ugly 😎